protocol/common/feature_flags.go

```diff
@@ -30,4 +30,8 @@ type FeatureFlags struct {
 	// AutoRequestHistoricMessages indicates whether we should automatically request
 	// historic messages on getting online, connecting to store node, etc.
 	AutoRequestHistoricMessages bool
+
+	// CuratedCommunitiesUpdateLoopEnabled indicates whether we should disable the curated communities update loop.
+	// Usually should be disabled in tests.
+	CuratedCommunitiesUpdateLoopEnabled bool
```

Comment:
Not sure if this belongs here. `CuratedCommunitiesUpdateLoop` is an implementation detail. It seems to be overkill to configure it with a feature flag :thinking:

Reply:
I agree that it doesn't sound like a "feature flag" at first. But it does enable some functionality of the app. And is `true` by default.

And otherwise we'd put it directly into config, which would make it a mess 😄